instrumentation/http-url-connection/javaagent/src/test/groovy/HttpUrlConnectionTest.groovy

```diff
@@ -157,7 +157,6 @@ class HttpUrlConnectionTest extends HttpClientTest<HttpURLConnection> implements
     runUnderTrace("someTrace") {
       HttpURLConnection connection = url.openConnection()
       connection.useCaches = useCaches
-      connection.addRequestProperty("is-test-server", "false")
```

Comment:
let's delete this whole test ("trace request without propagation") and rename the test above "trace request with propagation" -> "trace request"